### DISCUSSION TITLE
Reopening application of Github Campus expert!

### DISCUSSION THREAD

**Discussion by ritiksoni00** [community/community#23169] [New to GitHub]
<p>Can I get to know How much time it will take more to reopen the application of the Campus experts program?</p>
<p>P.S. it will be great if you provide the amount of time rather then “They will open them soon”</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Our Education Team are hard at work right now and there are changes on the way. More will be announced at <a href="https://githubuniverse.com/">Universe</a> in December.</p>

**Comment by ritiksoni00** [comment]
<p>Hey, <a class="mention" href="/canuckjacq">@canuckjacq</a> hope you are doing well there!</p>
<p>as I’m waiting for the application for the last 6 months the wait is over now!<br>
this will accelerate my college in the technology world!</p>
<p>I’m getting confused about some questions asked in the application form.<br>
like what if we don’t even have the community in our college…and the first question is asked about communities.?<br>
what should I write in 3rd part?<br>
<img src="https://user-images.githubusercontent.com/47344024/181075417-40cfcdfd-4b61-49a0-b3e0-caeeeb714e66.png" alt="image" data-base62-sha1="qWhLa14DPc8AYKrAwk2AnUB5lJp" width="535" height="152"></p>

**Comment by canuckjacq** [comment]
<p>:wave:</p>
<p>We don’t handle Education related queries at all here, I’m afraid, but if you re-read the Campus Expert website, maybe that will give you some clues about what they’re talking about:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://education.github.com/experts" target="_blank" rel="noopener">GitHub Education</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/11161530/181075420-3d179913-7830-403e-bf77-136cb92c5153.jpeg" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c99934504e0e9519775393ae88b99e667222517d_2_690x362.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c99934504e0e9519775393ae88b99e667222517d_2_1035x543.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/9/c99934504e0e9519775393ae88b99e667222517d.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c99934504e0e9519775393ae88b99e667222517d_2_10x10.png"></div>

<h3><a href="https://education.github.com/experts" target="_blank" rel="noopener">GitHub Campus Experts</a></h3>

<p>Technology leadership at your school. Become a Campus Expert to receive training in public speaking, community leadership, technical writing, and software development from GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I would assume they mean your student community that you’ll be working with as a campus expert.</p>
